packages/app/src/app/vscode/index.ts

```diff
@@ -74,9 +75,9 @@ class VSCodeManager {
         this.controller.getSignal('editor.toggleEditorPreviewLayout')();
       },
       keybindings: {
-        primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KEY_9,
+        primary: KeyMod.Shift || KeyMod.Alt || KeyCode.KEY_9,
```

Comment:
This has to be a `|`, keyboard shortcuts use binary shifting to determine the shortcut. If we use `||` it will break the shortcut. We should change our eslint rules for this.